zerver/lib/outgoing_webhook.py

```diff
@@ -157,6 +158,35 @@ def fail_with_message(event, failure_message):
     failure_message = "Failure! " + failure_message
     send_response_message(event['user_profile_id'], event['message'], failure_message)
 
+def get_message_url(event, request_data):
+    # type: (Dict[str, Any], Dict[str, Any]) -> Text
+    context = zulip_default_context(request_data)
+    message_url = ("%(server)s/#narrow/stream/%(stream)s/subject/%(subject)s/near/%(id)s"
+                   % {'server': context['realm_uri'],
+                      'stream': event['message']['display_recipient'],
+                      'subject': event['message']['subject'],
+                      'id': str(event['message']['id'])})
+    return message_url
+
+def notify_bot_owner(event, request_data, status_code=None, response_content=None, exception=None):
+    # type: (Dict[str, Any], Dict[str, Any], Optional[int], Optional[AnyStr], Optional[Any]) -> None
+    message_url = get_message_url(event, request_data)
+    bot_id = event['user_profile_id']
+    bot_user = get_user_profile_by_id(bot_id)
+    message_info = {'display_recipient': [{'email': bot_user.email},
+                                          {'email': bot_user.bot_owner.email}],
+                    'type': 'private'}
```

Comment:
we want the message to go the owner, right?  So shouldn't this be `bot_owner = bot_user.owner`, etc?

Reply:
I'm not sure if I understand you correctly; which part of this would you alter?

Reply:
I guess why is it copying `bot_user.email`?  That seems more likely than anything to create a message sending loop.